components/tree-select/tree-select.spec.ts

```diff
@@ -22,6 +31,38 @@ import { NzFormControlStatusType, NzFormModule } from 'ng-zorro-antd/form';
 import { NzTreeSelectComponent } from './tree-select.component';
 import { NzTreeSelectModule } from './tree-select.module';
 
+describe('nz-tree-select component', () => {
```

Comment:
suggestion: all the test are correct, but please move them into the basics section please